src/Registration/Models/Registration.php

```diff
@@ -0,0 +1,94 @@
+<?php declare(strict_types=1);
+
+namespace VSV\GVQ_API\Registration\Models;
+
+use Ramsey\Uuid\UuidInterface;
+use VSV\GVQ_API\User\Models\User;
+
+class Registration
+{
+    /**
+     * @var UuidInterface
+     */
+    private $id;
+    /**
+     * @var string
+     */
+    private $hashCode;
```

Comment:
@stejes we moeten hier best ook een value object voor maken i.p.v. gewoon een string. Misschien wel `PathSuffix` of `UniqueSuffix`